wfl/fit/mace.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
```

Comment:
I'd like to have a test for this functionality.  I guess it's a separate issue as to whether it can be run automatically by the github CI, but at least one that people can run manually.

Reply:
If I want to make pytest for this, pytest file should know where the `run_train.py` file is located. And this can be accessed by tracing back where the dist-info directory is. From there maybe I can look into `direct_url.json` file where it records the location of mace directory with `run_train.py` is located. So it can be done like this but I'm not totally sure whether this is proper way. Otherwise one has to specify where the path of `run_train.py` .

Reply:
You can use 
```
import mace
mace_dir = Path(mace.__file__).parent
```
for example, if you know that `run_train.py` is installed in the same directory as the mace python package (although it would be nicer if mace installed `run_train.py` as a proper binary, so it was just in your path, but obviously that's not something this PR can fix. I can try to convince them to do that).

If you write the pytest, I can help polish it so it runs cleanly on different machines.  Let me look into how the mace pip installation works.

Reply:
I've pushed a simple pytest for mace fitting. Please take a look and let me know if there's some room for improvement.

Reply:
Thanks.  I'll take a look, and see if I can improve the mace pip installer if that's needed as well.